shell/renderer/api/context_bridge/render_frame_context_bridge_store.h

```diff
@@ -58,11 +58,15 @@ class RenderFramePersistenceStore final : public content::RenderFrameObserver {
   // proxy maps are weak globals, i.e. these are not retained beyond
   // there normal JS lifetime.  You must check IsEmpty()
 
+  int32_t routing_id_;
```

Comment:
_(nit)_ This is set in the constructor and never changes, so might as well be `const`